bro-spinn/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/brospinn/SpinnService.kt

```diff
@@ -97,7 +88,7 @@ class SpinnService(
         val forespoerselId = Key.FORESPOERSEL_ID.les(UuidSerializer, melding)
         val eksternInntektsmelding = Key.EKSTERN_INNTEKTSMELDING.lesOrNull(EksternInntektsmelding.serializer(), melding)
         if (
-            eksternInntektsmelding?.avsenderSystemNavn != null &&
```

Comment:
Synes vi skal beholde denne sjekken også for sikkerhetsskyld.
Hvis avsender system er satt til null så ønsker vi ikke å sende det videre.
Evt så kan man gjøre den non nullable med default verdi til "Ukjent" eller noe slikt.

Reply:
Sjekken er bare erstattet med `eksternInntektsmelding != null`, som er funksjonelt identisk fordi `avsenderSystemNavn` har typen `String`, altså non-nullable.

Reply:
Jepp. Helt riktig 👍 
Hadde for meg at den var nullable men det er den jo ikke.